src/test/java/by/yurachel/web_app/servlet/AddPhoneServletTest.java

```diff
@@ -0,0 +1,82 @@
+package by.yurachel.web_app.servlet;
+
+import by.yurachel.web_app.entity.Phone;
+import by.yurachel.web_app.repository.PhoneRepository;
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+class AddPhoneServletTest {
+    private AddPhoneServlet addPhoneServlet;
+    private PhoneRepository pr;
+    private MockedStatic<PhoneRepository> capPhoneRep;
+
+    @BeforeEach
+    void setUp() {
+        addPhoneServlet = new AddPhoneServlet();
+        pr = mock(PhoneRepository.class);
+        capPhoneRep = Mockito.mockStatic(PhoneRepository.class);
+    }
+
+
+    @Test
+    void doGetTest() throws ServletException, IOException {
+        HttpServletRequest request = mock(HttpServletRequest.class);
```

Comment:
Reqeust/response/dispatcher mocks are duplicated in every method, move it to:
1) init method
2) use @Mock annotation for fields
3) create abstract test class